docs/queries/filters.md

```diff
@@ -134,6 +134,10 @@ As well as the date-related searches above, these filters search other propertie
 - `description (includes|does not include) <string>`
   - Matches case-insensitive (disregards capitalization).
   - Disregards the global filter when matching.
+- `description (regex matches|regex does not match) <JavaScript-style Regex>`
```

Comment:
Because I rushed getting the docs "done" to take my wife to the hardware store and completely forgot that we made these changes on a base class!

Reply:
😅

Reply:
> Because I rushed getting the docs "done" to take my wife to the hardware store and completely forgot that we made these changes on a base class!

Valid!!! 🤣

Reply:
The points mentioned above are all covered in my docs PR #1015, so resolving this conversation.